common/network-common/src/main/java/org/apache/spark/network/TransportContext.java

```diff
@@ -98,21 +98,32 @@ public TransportContext(
     this(conf, rpcHandler, closeIdleConnections, false);
   }
 
+    /**
+     *
+     * @param conf TransportConf
+     * @param rpcHandler RpcHandler responsible for handling requests and responses.
+     * @param closeIdleConnections Close idle connections if is set to true.
+     * @param isClientOnly This config is more important when external shuffle is enabled.
```

Comment:
I think for comments we follow the same spacing convention as observed here so sticking with it...